cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -27,27 +27,28 @@
 include(FindPackageHandleStandardArgs)
 include(SelectLibraryConfigurations)
 
-find_path(ZFP_INCLUDE_DIR
+find_path(ZFP_INCLUDE_DIRS
           NAMES zfp.h
-          PATHS $ENV{ZFP_ROOT}/inc
+          PATHS $ENV{ZFP_DIR}/include
```

Comment:
Why not including both paths (`inc` and `include`) here? was it broken in the first place?